packages/client/mutations/handlers/handleGitLabCreateIssue.ts

```diff
@@ -0,0 +1,53 @@
+import {parseWebPath} from './../../utils/parseWebPath'
+import {ConnectionHandler, RecordProxy, RecordSourceSelectorProxy} from 'relay-runtime'
+import createProxyRecord from '~/utils/relay/createProxyRecord'
+import toTeamMemberId from '../../utils/relay/toTeamMemberId'
+import {CreateTaskMutationResponse} from '../../__generated__/CreateTaskMutation.graphql'
+import getGitLabProjectsConn from '../connections/getGitLabProjectsConn'
+
+const handleGitLabCreateIssue = (
+  task: RecordProxy<NonNullable<CreateTaskMutationResponse['createTask']['task']>>,
+  store: RecordSourceSelectorProxy
+) => {
+  const integration = task.getLinkedRecord('integration')
+  if (!integration) return
+  const webPath = integration.getValue('webPath') as string | undefined
+  const {fullPath} = webPath ? parseWebPath(webPath) : {fullPath: ''}
+  const project = createProxyRecord(store, '_xGitLabProject', {
+    fullPath
+  })
+  if (!project) return
+  project.setLinkedRecords([integration], 'issues')
+  const teamId = task.getValue('teamId')
+  const meetingId = task.getValue('meetingId')
+  const viewer = store.getRoot().getLinkedRecord('viewer')
+  const viewerId = viewer?.getValue('id') as string
+  if (!viewerId || !meetingId) return
+  const teamMemberId = toTeamMemberId(teamId, viewerId)
+  const teamMember = store.get(teamMemberId)
+  const integrations = teamMember?.getLinkedRecord('integrations')
+  const gitlab = integrations
+    ?.getLinkedRecord('gitlab')
+    ?.getLinkedRecord('api')
+    ?.getLinkedRecord('query')
+  const typename = integration.getType()
+  if (typename !== '_xGitLabIssue') return
+  const gitlabProjectsConn = getGitLabProjectsConn(gitlab) // TODO: add query string
+  if (!gitlabProjectsConn) return
+  const now = new Date().toISOString()
+  const newEdge = ConnectionHandler.createEdge(
+    store,
+    gitlabProjectsConn,
+    project,
+    'GitLabProjectEdge'
+  )
+  const mysteryIssueInEdge = newEdge!
+    .getLinkedRecord('node')
+    ?.getLinkedRecords('issues')![0]
+    ?.getValue('id')
+  console.log('🚀 ', {mysteryIssueInEdge})
+  newEdge.setValue(now, 'cursor')
+  ConnectionHandler.insertEdgeBefore(gitlabProjectsConn, newEdge)
```

Comment:
Hey @mattkrick, in the GitLab [scoping results](https://github.com/ParabolInc/parabol/blob/79ac25607102e774cb2339dbc0725dcfb4fafbf6/packages/client/components/GitLabScopingSearchResults.tsx#L97), we query the projects and then grab the issues within those projects. This differs from the other integrations where we just query the issues.

When creating a new issue, I want to insert a new edge in the connection. However, we can't have an issues connection that is a child of the projects connection. So, I want the new edge to be a project that contains the new issue. 

I can see here that the new project edge does contain the new issue. I can see it in the Relay dev tools too. However, when you add a new issue, you'll see in this [console log](https://github.com/ParabolInc/parabol/blob/79ac25607102e774cb2339dbc0725dcfb4fafbf6/packages/client/components/GitLabScopingSearchResults.tsx#L139) that a new project edge has been added but it does not contain the new issue.

I'm wondering if you might know why that it and if there's a better way of inserting the issue?